lib/github.js

```diff
@@ -71,6 +71,12 @@ Organization.prototype.getRepositories = function(callback) {
             cli.debug('next page ' + this.nextPageUrl);
             cli.debug('last page ' + this.getLastPage());
 
+            if ( body.message ) {
```

Comment:
Just tested here, tried to update the lib but no luck. Looks like what you did checking `response.statusCode` is the best solution. BTW, this block of code isn't need anymore, right?